src/khoj/routers/api_chat.py

```diff
@@ -583,6 +586,41 @@ async def send_rate_limit_message(message: str):
             await conversation_command_rate_limiter.update_and_check_if_valid(websocket, cmd)
             q = q.replace(f"/{cmd.value}", "").strip()
 
+        if conversation_commands == [ConversationCommand.Summarize]:
```

Comment:
We should also add this to the tool selection process so it can be dynamically inferred.